### PR TITLE
Add role bindings and cluster role bindings, organize groups and improve the CRD names

### PR DESCRIPTION
**Pull request by maciaszczykm**
Related to https://github.com/kubernetes/dashboard/issues/5232.

### CODE REVIEW

File: src/app/frontend/resource/cluster/rolebinding/detail/component.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import {EndpointManager, Resource} from '../../../../common/services/resource/endpoint';
+import {NamespacedResourceService} from '../../../../common/services/resource/resource';
+import {KdStateService} from '../../../../common/services/global/state';
+
+@Component({
+  selector: 'kd-role-detail',
+  templateUrl: './template.html',
+})
+export class RoleBingingDetailComponent implements OnInit, OnDestroy {
+  private roleSubscription_: Subscription;
```
Comment (floreks) [inline src/app/frontend/resource/cluster/rolebinding/detail/component.ts:31]: Do not use the old way of managing subscriptions. Use `_unsubscribe` Subject.

File: src/app/frontend/resource/cluster/clusterrolebinding/detail/component.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import {EndpointManager, Resource} from '../../../../common/services/resource/endpoint';
+import {ResourceService} from '../../../../common/services/resource/resource';
+import {KdStateService} from '../../../../common/services/global/state';
+
+@Component({
+  selector: 'kd-cluster-role-binding-detail',
+  templateUrl: './template.html',
+})
+export class ClusterRoleBindingDetailComponent implements OnInit, OnDestroy {
+  private clusterRoleSubscription_: Subscription;
```
Comment (floreks) [inline src/app/frontend/resource/cluster/clusterrolebinding/detail/component.ts:31]: Same here.

**Comment by maciaszczykm** [comment]
PTAL

**Pull request merged**